cluster-up/cluster/kind/common.sh

```diff
@@ -15,6 +15,8 @@ REGISTRY_NAME=${CLUSTER_NAME}-registry
 MASTER_NODES_PATTERN="control-plane"
 WORKER_NODES_PATTERN="worker"
 
+ETCD_DATA_DIR=${ETCD_DATA_DIR:-none}
```

Comment:
You don't need this, just check if the variable set or not before use it
```bash
[  -z "$ETCD_DATA_DIR" ]
```

Reply:
I added this to common.sh for better visibility of that variable, so if one will review for example `ensure_memory_dir_for_etcd_data` function it will be easier to understand from where this variable comes form
What you think?

Reply:
If you want to do that, then just throw an error here is the variable is not set.